templates/index.html

```diff
@@ -164,28 +171,29 @@ <h4 class="panel-title">
                 </div>
             </div>
             
-            <br/>
-            <button class="btn btn-success" id="details">
-                View Details</button> 
-            <button class="btn btn-success" id="export">Export Data</button>
-            <br/>(Only supported in Accident Mode)
+            <!--<br/>(Only supported in Accident Mode)-->
             
         </div>
         <div class="col-md-9">
 
             <div class="row">
-                <div class="col-md-6">	
+                <div class="col-md-3">	
                     <div id="userinfo"></div>
                 </div>
-                <div class="col-md-6">
+                <div class="col-md-4">	
+                    <button class="btn btn-success" id="details">
+                        View Details</button> 
+                    <button class="btn btn-success" id="export">Export Data</button>
+                </div>
+                <div class="col-md-5">
                     <button class="btn btn-success btn-pressure pull-right" id="roadsegments">RoadSegments</button>
```

Comment:
When all the facet filtering panel are expanded, these buttons will be out of screen.